README.md

```diff
@@ -225,7 +225,7 @@ An example of using `fastify-swagger` with `static` mode enabled can be found [h
  | staticCSP          | false            | Enable CSP header for static resources.                                                                                   |
  | stripBasePath      | true             | Strips base path from routes in docs.                                                                                     |
  | swagger            | {}               | [Swagger configuration](https://swagger.io/specification/v2/#swaggerObject).                                              |
- | transform          | null             | Transform method for schema.                                                                                              |
+ | transform          | null             | Transform method for the route's schema and url: `(schema: any, url: string) => { schema: any, url: string }`.                                                                                              |
```

Comment:
**IMPORTANT: this is a breaking change that requires a major version release**

it changes the function return value.
so any existing implementations will have to be migrated from returning simple `return transformed` to `return { schema: transformed, url }`

Reply:
we can add some fallback to support the old code (and release as minor), but it'l be a little ugly.
fallback code is usually ugly  😆

Reply:
I would use a single object for the parameter.
`{ schema, url } => { schema, url }`

Reply:
done, but now its even a bigger breaking change :) need to release a BIG major

Reply:
It is OK, the current syntax is much extendable in future.